code/addons/docs/package.json

```diff
@@ -120,6 +119,7 @@
     "ts-dedent": "^2.0.0"
   },
   "devDependencies": {
+    "@mdx-js/mdx": "^3.0.0",
```

Comment:
Can you elaborate on why prebundling is necessary here?

Reply:
because of ESM only issues. The prebundling generates CJS compatible code